packages/graphql-codegen-supermassive-typed-document-node-plugin/README.md

```diff
@@ -0,0 +1,5 @@
+# Custom GraphQL codegen plugins
+
+## supermassive-typed-document-node
+
+Do the same stuff as the original typed-document-node plugin except it extends GraphQL AST data to match Supermassive AST
```

Comment:
Always try to reduce work on the reader and include links to references. i.e. the original plugin and perhaps supermassive AST details.